src/actions/deleteUserActions.js

```diff
@@ -1,33 +1,46 @@
-const fs = require('fs')
-
-let findUser = (users, id) => {
-    return users.find(e => e.id == id)
-}
+const fs = require("fs");
+const { readFile, writeFile, findItemById } = require("../helpers/helpers");
```

Comment:
You may need to split the helpers into file helpers and database helpers at some point